src/elm/Session/LoggedIn.elm

```diff
@@ -172,6 +187,9 @@ initModel shared authModel accountName selectedCommunity =
     , hasShop = FeatureLoading
     , hasObjectives = FeatureLoading
     , hasKyc = FeatureLoading
+    , searchModel = Search.init selectedCommunity
+    , claimingAction = { status = Action.NotAsked, feedback = Nothing, needsPinConfirmation = False }
```

Comment:
feedback here isn't duplicating the logged in feedback? i know it's needed but i would love a deeper explanation 🙃

Reply:
It's not the duplication but more like the preparation of the messages for the feedback. I went that way because I couldn't import `LoggedIn.elm` into `Action.elm` (it will cause the circular dependency) but I wanted to handle all the stuff related to claiming inside the `Action` module to keep `LoggedIn` less bloated. So I'm preparing the messages for the feedback inside `Action.elm` and then use them in `LoggedIn.elm` to show.